client/sites/components/sites-dashboard.tsx

```diff
@@ -193,44 +192,47 @@ const SitesDashboard = ( {
 				},
 			},
 		},
-	} as DataViewsState;
-	const [ dataViewsState, setDataViewsState ] = useState< DataViewsState >( defaultDataViewsState );
+	} as View;
+	const [ viewState, setViewState ] = useState< View >( defaultViewState );
 
 	useEffect( () => {
 		const fields = getFieldsByBreakpoint( isDesktop );
 		const fieldsForBreakpoint = [ ...fields ].sort().toString();
-		const existingFields = [ ...( dataViewsState?.fields ?? [] ) ].sort().toString();
+		const existingFields = [ ...( viewState?.fields ?? [] ) ].sort().toString();
 		// Compare the content of the arrays, not its referrences that will always be different.
 		// sort() sorts the array in place, so we need to clone them first.
 		if ( existingFields !== fieldsForBreakpoint ) {
-			setDataViewsState( ( prevState ) => ( { ...prevState, fields } ) );
+			setViewState( ( prevState ) => ( { ...prevState, fields } ) );
```

Comment:
Considering rolling these prop/state/variables renames back as they're adding a lot of noise to an already big change. `viewState` should probably be `view` too now that I see it's just the `view` prop on DataView`.

Reply:
I think this will help reviewers a lot 👍

Reply:
90c821ea94840e8749b12328cd3d4dc28371470c